entities/EmailClient.js

```diff
@@ -0,0 +1,19 @@
+const { formatBody } = require('../lib/utilities/formatBody');
+const { sendMail } = require('../lib/channels/sendMail');
+
+class EmailClient {
+  constructor(destination, results) {
+    this.destination = destination;
+    this.message = formatBody(results);
+  }
+
+  async send() {
+    const result = await sendMail(this.destination, this.message);
```

Comment:
I might have included the `sendMail` method (and its dependencies) as static methods in `EmailClient` (or created a class `EmailSender` that contains those methods that gets "injected" into `EmailClient`), but I think the way you've done it here is clear, so not a big deal either way

Reply:
I'm going to pass on this one for now